client/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:1.16
```

Comment:
is this file a leftover? Doesn't seem referenced from the `docker-compose` YAML configuration